lib/streamlit/elements/heading.py

```diff
@@ -54,9 +57,10 @@ def header(
               where ``color`` needs to be replaced with any of the following
               supported colors: blue, green, orange, red, violet.
 
-        anchor : str
+        anchor : str or False
```

Comment:
Will empty string or None be interpreted as `anchor=False`? should it? In my opinion it should, otherwise LGTM 👍

Reply:
`None` means that the value should be generated automatically. It is covered by docstring.

> If omitted, it generates an anchor using the body.

Reply:
Got it 👍